website_hr_department/__openerp__.py

```diff
@@ -26,10 +28,11 @@
     'summary': """
         Display the structure of your departments and their members.
         """,
-    'author': 'ACSONE SA/NV, '
-            'Odoo Community Association (OCA). '
-            'Serpent Consulting Services Pvt. Ltd.',
-    'website': "http://acsone.eu,serpentcs.com",
+    'author': "ACSONE SA/NV, "
+              "Odoo Community Association (OCA), "
+              "Serpent Consulting Services Pvt. Ltd. ",
+             
+    'website': "http://acsone.eu, http://www.serpentcs.com",
```

Comment:
If you put several websites, none will work because this is taken as a single URL. Not a blocker though.